freezeyt/cli.py

```diff
@@ -9,7 +9,8 @@
 @click.argument('module_name')
 @click.argument('dest_path')
 @click.option('--prefix', default='http://localhost:8000/', help='URL of the application root')
-def main(module_name, dest_path, prefix):
+@click.option('--extra-pages', multiple=True, help='Pages without any link in application')
```

Comment:
This is used as `python -m freezeyt --extra-pages page1 --extra-pages page2 ...`, right?
`extra-page` would be a better name.

Reply:
I've renamed.